src/visualization/types/Graph/options.tsx

```diff
@@ -48,6 +55,10 @@ const GraphViewOptions: FC<Props> = ({properties, results, update}) => {
     return columnType === 'time' || columnType === 'number'
   })
 
+  const groupKey = useMemo(() => [...results.fluxGroupKeyUnion, 'result'], [
```

Comment:
Because this is a hook, i could not put it behind a feature flag, rest of the logic is behind the `graphColorMapping` flag.